main.nf

```diff
@@ -1083,6 +1083,7 @@ process proteomicslfq {
      file "out.mzTab" into out_mztab_plfq, out_mztab_msstats
      file "out.consensusXML" into out_consensusXML
      file "out.csv" optional true into out_msstats
```

Comment:
@jpfeuffer would be good to add the prefix of the names of the tools like: 

- out_mstats.csv 
- out_triqler.tsv 

In this way, the user know what is the output file format.